<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1146</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="3">
        <widget class="QWidget" name="wd_horizon_attitude" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>380</width>
           <height>200</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>380</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QWidget" name="wd_controls" native="true">
         <property name="maximumSize">
          <size>
           <width>450</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>350</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>350</width>
              <height>0</height>
             </size>
            </property>
            <property name="title">
             <string>Command Console</string>
            </property>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Operational Mode</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="lbl_status_operational_mode">
                 <property name="text">
                  <string>?</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="dd_operational_mode"/>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Lidar Recording</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="lbl_status_lidar_recording">
                 <property name="text">
                  <string>?</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_enable_lidar_recording">
                 <property name="text">
                  <string>Enable</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_disable_lidar_recording">
                 <property name="text">
                  <string>Disable</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>App Video</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QPushButton" name="btn_enable_app_video">
                 <property name="text">
                  <string>Enable</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_disable_app_video">
                 <property name="text">
                  <string>Disable</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QWidget" name="wd_status" native="true">
         <property name="maximumSize">
          <size>
           <width>450</width>
           <height>250</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>250</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="tabPosition">
             <enum>QTabWidget::South</enum>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <property name="elideMode">
             <enum>Qt::ElideNone</enum>
            </property>
            <property name="documentMode">
             <bool>true</bool>
            </property>
            <property name="tabBarAutoHide">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="tabSystemStatus">
             <attribute name="title">
              <string>System Status</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <widget class="QWidget" name="widget_4" native="true">
                <layout class="QFormLayout" name="formLayout_3">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>Last updated</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLabel" name="lbl_status_last_updated">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>CPU Load</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1">
                  <widget class="QLabel" name="lbl_status_cpu_load">
                   <property name="text">
                    <string>0.0 0.0 0.0</string>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="0">
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>RAM / Swap</string>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="1">
                  <widget class="QLabel" name="lbl_status_memory">
                   <property name="text">
                    <string>348M / 4192M</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tabGpsDetails">
             <attribute name="title">
              <string>GPS Details</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_10">
              <item>
               <widget class="QWidget" name="widget_5" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_9">
                 <item>
                  <widget class="QWidget" name="widget_2" native="true">
                   <layout class="QFormLayout" name="formLayout">
                    <item row="0" column="0">
                     <widget class="QLabel" name="label_11">
                      <property name="text">
                       <string>Lat</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QLabel" name="lbl_status_lat">
                      <property name="text">
                       <string>TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0">
                     <widget class="QLabel" name="label_15">
                      <property name="text">
                       <string>Lon</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="1">
                     <widget class="QLabel" name="lbl_status_lon">
                      <property name="text">
                       <string>TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="0">
                     <widget class="QLabel" name="label_17">
                      <property name="text">
                       <string>HDOP</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="1">
                     <widget class="QLabel" name="lbl_status_hdop">
                      <property name="text">
                       <string>TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="0">
                     <widget class="QLabel" name="label_19">
                      <property name="text">
                       <string>VDOP</string>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="1">
                     <widget class="QLabel" name="lbl_status_vdop">
                      <property name="text">
                       <string>TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="4" column="0">
                     <widget class="QLabel" name="label_21">
                      <property name="text">
                       <string>PDOP</string>
                      </property>
                     </widget>
                    </item>
                    <item row="4" column="1">
                     <widget class="QLabel" name="lbl_status_pdop">
                      <property name="text">
                       <string>TextLabel</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QWidget" name="widget_3" native="true">
                   <layout class="QFormLayout" name="formLayout_4">
                    <item row="0" column="0">
                     <widget class="QLabel" name="label_12">
                      <property name="text">
                       <string>#SAT Visible</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QLabel" name="lbl_status_sat_visible">
                      <property name="text">
                       <string>TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0">
                     <widget class="QLabel" name="label_23">
                      <property name="text">
                       <string>#SAT used</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="1">
                     <widget class="QLabel" name="lbl_status_sat_used">
                      <property name="text">
                       <string>TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="0">
                     <widget class="QLabel" name="label_25">
                      <property name="text">
                       <string>Pressure</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="1">
                     <widget class="QLabel" name="lbl_status_pressure">
                      <property name="text">
                       <string>TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="0">
                     <widget class="QLabel" name="label_27">
                      <property name="text">
                       <string>Altitude</string>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="1">
                     <widget class="QLabel" name="lbl_status_altitude">
                      <property name="text">
                       <string>TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="4" column="0">
                     <widget class="QLabel" name="label_29">
                      <property name="text">
                       <string>Temperature</string>
                      </property>
                     </widget>
                    </item>
                    <item row="4" column="1">
                     <widget class="QLabel" name="lbl_status_temperature">
                      <property name="text">
                       <string>TextLabel</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" rowspan="2">
        <widget class="QWidget" name="wd_lidar" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>600</width>
           <height>600</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>600</width>
           <height>600</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="lbl_lidar_map">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QWidget" name="wd_planner" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_8"/>
        </widget>
       </item>
       <item row="0" column="2" rowspan="2" colspan="3">
        <widget class="QWidget" name="wd_video" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QComboBox" name="dd_stream"/>
            </item>
            <item>
             <widget class="QToolButton" name="btn_play">
              <property name="text">
               <string>Play</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btn_stop">
              <property name="text">
               <string>Stop</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1146</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
